sci-physics/siscone/siscone-3.0.3.ebuild

```diff
@@ -0,0 +1,28 @@
+# Copyright 1999-2016 Gentoo Foundation
+# Distributed under the terms of the GNU General Public License v2
+# $Id$
+
+EAPI=6
+
+DESCRIPTION="Hadron Seedless Infrared-Safe Cone jet algorithm"
+HOMEPAGE="http://siscone.hepforge.org/"
+SRC_URI="http://www.hepforge.org/archive/${PN}/${P}.tar.gz"
+
+LICENSE="GPL-2"
+SLOT="0"
+KEYWORDS="~amd64 ~x86 ~amd64-linux ~x86-linux"
+IUSE="examples static-libs"
```

Comment:
sounds like an `autotools-utils` conversion and you forgot to tack the `use_enable` static-libs back on?